src/main/java/be/polyscripts/contactmanagerapp/service/CompanyService.java

```diff
@@ -0,0 +1,46 @@
+package be.polyscripts.contactmanagerapp.service;
+
+
+import be.polyscripts.contactmanagerapp.exceptions.CompanyNotFoundException;
+import be.polyscripts.contactmanagerapp.model.Company;
+import be.polyscripts.contactmanagerapp.repo.CompanyRepository;
+import be.polyscripts.contactmanagerapp.ressource.ContactResource;
+import lombok.RequiredArgsConstructor;
+import org.hibernate.annotations.common.util.impl.LoggerFactory;
+import org.jboss.logging.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+@Transactional

```

Comment:
why Transactionnal if you don't need Rollbacks ?

Reply:
Yes, for now 😅